ui/packages/consul-ui/vendor/consul-ui/routes.js

```diff
@@ -7,32 +7,32 @@
       index: {
         _options: {
           path: '/',
-          redirect: '../show/serverstatus',
+          redirect: '../services',
```

Comment:
Confirming that this reverts to the previous behavior for `/`, as shown here: https://github.com/hashicorp/consul/commit/4b8cabce0df97c4a396c4b843dc5fe46d2c40c92#diff-19c54e426e573c20da7516c4e1e112af0d215ee3dc968f2b18d58d9030c47ebfR10